### ISSUE TITLE
global redir

### ISSUE THREAD

**Issue by virtimus** [Apicurio/apicurio-docker#3]
Currently after running the image and trying to access app at localhost goes global redir to https://auth-apistudio.rhcloud.com/auth/realms/apicurio-local/protocol/openid-connect/auth?response_type=code&client_id=apicurio-studio&redirect_uri=http%3A%2F%2Fi3c.h%3A8081%2Fstudio%2F&state=f2257087-81e4-42e3-b6f0-cf459fe83902&login=true&scope=openid

**Comment by EricWittmann** [comment]
Yeah apologies for that - the combined docker image is a bit stale.  More recently we created an openshift template and docker images for the individual pieces.  The all-in-one docker image is stale and has been temporarily removed from the project site.  I'm hoping to find some time soon to freshen it back up.

That said, the main problem with that docker image is that it's just running an old version of the quickstart which is configured to use a cloud authentication service that is no longer available.  If you were to rebuild the docker image with the most recent version of apicurio it would likely work.  Or wait for us to do it.  :)

Another alternative, of course, is to download the quickstart zip and use that directly (rather than use docker).
